lib/PGcore.pm

```diff
@@ -549,6 +552,18 @@ sub store_persistent_data {    # will store strings only (so far)
 	$label;
 }
 
+sub update_persistent_data {    # will store strings only (so far)
+	my $self  = shift;
+	my $label = shift;
+	$self->{PERSISTENCE_HASH}->{$label} = join("", @_);
+}
+
+sub get_persistent_data {
+	my $self  = shift;
+	my $label = shift;
+	return ($self->{PERSISTENCE_HASH}->{$label});
```

Comment:
Similar changes here.  This should be
```suggestion
	my ($self, $label) = @_;
	return $self->{PERSISTENCE_HASH}{$label};
```
perlcritic won't complain about this, but it is still better as it makes the expected arguments clear.  Note there is no need for the parenthesis around the return value.  The parenthesis don't add anything.  Note that parenthesis do not turn values into lists or arrays.  The calling context does that.